doc/sphinx/proof-engine/ltac2.rst

```diff
@@ -464,14 +468,25 @@ Generic Syntax for Quotations
 In general, quotations can be introduced in terms using the following syntax, where
 :production:`quotentry` is some parsing entry.
```

Comment:
This sentence does not make sense without the `prodn` that was deleted below. It should be adjusted.